dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -564,6 +571,9 @@
             <SolidColorBrush x:Key="SystemFillColorCriticalBackgroundBrush" Color="{ThemeResource SystemColorWindowColor}" />
             <SolidColorBrush x:Key="SystemFillColorNeutralBackgroundBrush" Color="{ThemeResource SystemColorWindowColor}" />
 
+            <SolidColorBrush x:Key="SystemControlFocusVisualPrimaryBrush" Color="{StaticResource SystemColorWindowTextColor }"/>
```

Comment:
You should move these changes into Common_themeresources.xaml. The _any is included in both version1 and version2 styles.

Reply:
Done now.